pkgs/racket-doc/scribblings/guide/other-editors.scrbl

```diff
@@ -284,3 +284,39 @@ provides support for syntax highlighting and building for Sublime Text.
 
 The @hyperlink["https://marketplace.visualstudio.com/items?itemName=evzen-wybitul.magic-racket"]{Magic Racket}
 extension provides Racket support including REPL integration and syntax highlighting in Visual Studio Code.
+
+@section{racket-langserver}
+
+The @tt{racket-langserver} is a @hyperlink["http://langserver.org/"]{Language Server Protocol}
+implementation for Racket. This project seeks to use @hyperlink["https://github.com/racket/drracket"]{DrRacket's public APIs} 
```

Comment:
If you want a link here, I think it should be `@seclink[#:indirect? #t #:doc '(lib "scribblings/drracket-tools/drracket-tools.scrbl") "top"]{DrRacket's public APIs}`.